static/index.html

```diff
@@ -47,6 +47,8 @@
   Note: #! has never provided user data to any government agency.
         Pay close attention to the removal of this notice.
 
+  All code licensed via MIT: https://hashbang.sh/static/LICENSE.txt
```

Comment:
Actually since you are using the markdown version with the correct mimetype s/txt/md/g